Meta/run.sh

```diff
@@ -72,10 +72,8 @@ if [ -z "$SERENITY_QEMU_BIN" ]; then
     fi
     if [ "$SERENITY_ARCH" = "aarch64" ]; then
         SERENITY_QEMU_BIN="${QEMU_BINARY_PREFIX}qemu-system-aarch64${QEMU_BINARY_SUFFIX}"
-    elif [ "$SERENITY_ARCH" = "x86_64" ]; then
+    else [ "$SERENITY_ARCH" = "x86_64" ]
```

Comment:
Is this correct bash syntax? if we don't have any other checking of $SERENITY_ARCH, might be good to keep the else and error out